spec.html

```diff
@@ -49421,15 +49438,12 @@ <h1>
       <h1>
         HostEventSet (
           _execution_: a candidate execution,
-        ): a Set of events
+        ): a Set of Memory events
       </h1>
       <dl class="header">
       </dl>
       <emu-alg>
-        1. Let _events_ be an empty Set.
-        1. For each event _E_ of EventSet(_execution_), do
-          1. If _E_ is not in SharedDataBlockEventSet(_execution_), add _E_ to _events_.
-        1. Return _events_.
+        1. Return the subtraction of SharedDataBlockEventSet(_execution_) from EventSet(_execution_).
```

Comment:
I'd prefer not to rely on this novel phrasing. It's just as easy to be explicit about it.

```suggestion
        1. Return a new Set that contains all elements of EventSet(_execution_) that are not in SharedDataBlockEventSet(_execution_).
```

Reply:
```suggestion
        1. Return a new Set containing all elements of EventSet(_execution_) that are not in SharedDataBlockEventSet(_execution_).
```

Reply:
WFM

Reply:
What do you dislike about subtraction @michaelficarra?

Reply:
I don't dislike it at all, but if we did use it, I'd at least want to say somewhere in the Set section that this is a thing we can do (and possibly explicitly define it). Instead though, it's basically just as easy to write it out.

Reply:
Oh, it already does say that: [The Set and Relation Specification Types](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-set-and-relation-specification-type) (emphasis mine):
> Sets may be unioned, intersected, or **subtracted** from each other.

Reply:
Oh! I didn't know that was there. In that case, yeah, using subtraction is fine.